storybook/stories/materials/icons/icons.stories.mdx

```diff
@@ -116,6 +117,20 @@ As a background image:
 
 </details>
 
+## Usage
+
+Icons provided inherently do not provide a width and height property nor a colour, to favour reusability. This means that you need to provide dimensions, either using the width and height property or using CSS. If you are using [Tachyons](/?path=/docs/introduction-tachyons--page) this means using these classes: _w6 h6_, for a 24x24 icon.
```

Comment:
> Icons provided inherently do not provide a width and height property nor a colour, to favour reusability. This means that you need to provide dimensions, either using the width and height property or using CSS. If you are using [Tachyons](/?path=/docs/introduction-tachyons--page) this means using these classes: _w6 h6_, for a 24x24 icon.

Proposition:

Icons does not have a default dimensions to favour reusability. This means that you must specify dimensions either with [Tachyons](/?path=/docs/introduction-tachyons--page) dimensions classes (e.g. `w6 h6`) or straight CSS.